bin/benchpark

```diff
@@ -261,8 +263,6 @@ def benchpark_setup_handler(args):
     instructions = f"""\
 To complete the benchpark setup, do the following:
 
-    cd {workspace_dir}/workspace
-
```

Comment:
Without this, the later suggestions `ramble -P -D . workspace setup` and `ramble -P -D . on` are possibly confusing (they wouldnt work until you `cd` into the benchmark workspace dir): you could update them (e.g. `ramble -P -D {ramble_workspace_dir} workspace setup`)

Reply:
The cd is not required for the spack and ramble commands that have the full path.

But, I was wondering what we should have here to help the user with the next steps.  ramble_workspace_dir will be set to a new value with each experiment the user adds to this benchpark_workspace (which we might want to always call benchpark_workspace for clarity), so when these instructions are displayed, they will point at the ramble_workspace_dir for the last experiment the user added.  Any suggestions on how to communicate that:
- There is a ramble_workspace_dir per experiment
- That's what the user should cd into to work with the specific experiment